### DISCUSSION TITLE
是否有人愿意参与合并分支等？可以在此留言，给权限，合并新的提交以及校验数据是否正确，其他切勿修改

### DISCUSSION THREAD

**Discussion by WeNeedHome** [WeNeedHome/SummaryOfLoanSuspension#423] [Ideas]
是否有人愿意参与合并分支等？可以在此留言，给权限，合并新的提交以及校验数据是否正确，其他切勿修改

首先你github总的玩的溜啊，别把文档合并坏了

**Comment by WeNeedHome** [comment]
会git，懂merge的，愿意的，不发过激言论的

**Reply by Maksim-venus** [reply]
不会pull，增加济南阳光城檀悦图片[济南阳光城檀悦](https://github.com/Maksim-venus/SummaryOfLoanSuspension/blob/main/images/%E6%B5%8E%E5%8D%97%E9%98%B3%E5%85%89%E5%9F%8E%E6%AA%80%E6%82%A6.jpeg)

**Reply by taopeng-life** [reply]
就是玩的不6，提个PR研究半天

**Reply by TwilightT** [reply]
您好，广西玉林那个是中鼎绿城中心，不是中鼎绿地中心

**Reply by yalo-ddaa** [reply]
楼主您好。我的家人也曾经是烂尾楼的受害者之一，非常感谢您的项目，希望能帮上忙。
经常用git，近期也没什么事，都可以帮忙。
项目的pr积攒的还是比较多的，我可以帮忙把未完善好的pr修改一下并提交合并。

**Reply by 175536** [reply]
桂林融创N7地块的图不对，是N4地块的，我提供N7地块的图在下面，麻烦更新
![图片](https://user-images.githubusercontent.com/96563305/179187470-af13c247-0195-4369-9f88-d68d96d85730.png)

**Reply by yalo-ddaa** [reply]
> 桂林融创N7地块的图不对，是N4地块的，我提供N7地块的图在下面，麻烦更新 ![图片](https://user-images.githubusercontent.com/96563305/179187470-af13c247-0195-4369-9f88-d68d96d85730.png)

我可以帮忙发一下pr

**Reply by 175536** [reply]
再提供一个桂林融创文旅城N5地块（11期）的清晰版
![图片](https://user-images.githubusercontent.com/96563305/179189686-5ee26da3-c459-491b-9db6-828d8cbf178c.png)

**Comment by middzwb** [comment]
我可以，不过工作日要晚上10点之后才能更新。自己也是郑州期房业主

**Reply by LiuJinfan** [reply]
兄弟 少加点班 身体要紧
唉 现在的人民太苦了

**Reply by WeNeedHome** [reply]
给了

**Reply by ljw-bigtail** [reply]
层主可以给合下pr吗#256 提供图了

**Comment by f2h2h1** [comment]
我愿意，会git，懂merge，不发表言论

**Reply by sharkwuu** [reply]
吼啊

**Reply by WeNeedHome** [reply]
> 我愿意，会git，懂merge，不发表言论

给了

**Reply by feodorren** [reply]
请帮忙提交 #463 ，谢谢

**Comment by oceanzhang860411** [comment]
請務必保證，不要被粉紅，或者國安混進來，否則直接給你刪了

**Reply by sharkwuu** [reply]
说的对，务必谨慎!

**Comment by frankxjkuang** [comment]
我愿意，尽微薄之力

**Reply by WeNeedHome** [reply]
给了

**Reply by BeefFat** [reply]
加一下 武汉市黄陂区恒大龙城

**Comment by BlueWhale235** [comment]
我愿意，会一些git，懂merge

**Comment by darvincy** [comment]


**Reply by tuke0919** [reply]
大家及时fork，备份到自己的仓库

**Comment by jxnu-liguobin** [comment]
容易被坏人混进来

**Comment by dudebing99** [comment]
1 愿意协助处理 PR

**Comment by killdayu** [comment]
我愿意，会pr

**Comment by LYM1029** [comment]
我会git操作，晚上有时间更新，南昌烂尾楼业主

**Reply by x147369** [reply]
南昌新力还是恒大啊

**Comment by wongzhe** [comment]
希望您帮忙添加一下，天津市 北辰区 融创御景宸院项目




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "WeNeedHome/SummaryOfLoanSuspension"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年7月15日(星期五) 中午11:15
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [WeNeedHome/SummaryOfLoanSuspension] 是否有人愿意参与合并分支等？可以在此留言，给权限，合并新的提交以及校验数据是否正确，其他切勿修改 (Discussion #423)





 
我愿意，会pr

**Comment by LYM1029** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by wongzhe** [comment]
希望您帮忙添加一下 天津市 北辰区 融创御景宸院项目




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "WeNeedHome/SummaryOfLoanSuspension"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年7月15日(星期五) 中午11:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [WeNeedHome/SummaryOfLoanSuspension] 是否有人愿意参与合并分支等？可以在此留言，给权限，合并新的提交以及校验数据是否正确，其他切勿修改 (Discussion #423)





 
1

**Reply by WeihanLi** [reply]
有图片或者链接之类的证明材料吗

**Comment by kujiji-321** [comment]
我可以帮忙，懂git会merge，不发表言论
中午有时间

**Reply by hunao0221** [reply]
我可以，白天有时间

**Comment by silfoxs** [comment]
我可以帮忙处理PR我是专业的

**Comment by wongzhe** [comment]
谢谢您了




------------------ 原始邮件 ------------------
发件人:                                                                                                                        "WeNeedHome/SummaryOfLoanSuspension"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年7月15日(星期五) 中午11:24
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [WeNeedHome/SummaryOfLoanSuspension] 是否有人愿意参与合并分支等？可以在此留言，给权限，合并新的提交以及校验数据是否正确，其他切勿修改 (Discussion #423)





 
我可以帮忙，懂git会merge，不发表言论
 中午有时间

**Comment by pfnsnp** [comment]
如果要 pull request 请按照格式

    1、修改总数以及添加位置的数量
    2、最好有图片

大家都修改数量，都修改的话那不就乱套了和多线程去操作同一个变量一样。  总数和位置数量只能合并pr时候去修改。 
@WeNeedHome 
 希望修改下规则，让合并人员负责修改。

**Comment by GuoyiZhang** [comment]
我可以吗？

**Comment by GuoyiZhang** [comment]
我可以吗？

**Comment by WeihanLi** [comment]
可以+1

**Comment by trunklater** [comment]
有时间，会合并 +1

**Comment by JoannaHan2022** [comment]
麻烦帮忙添加一下  湖北省宜昌市夷陵区恒大名都项目  感谢🙏

**Reply by WeihanLi** [reply]
有图片或者链接吗

**Reply by WeihanLi** [reply]
是这个吗 https://github.com/WeNeedHome/SummaryOfLoanSuspension/pull/269

**Comment by dalezhang93** [comment]
可以的，git用的很熟

**Comment by yalo-ddaa** [comment]
楼主辛苦了，我的家人也曾经是烂尾楼的受害者之一，希望能帮忙。
经常用git，近期也没什么事，都可以帮忙。
项目的处理速度还是比较慢的，我可以帮忙把未完善好的pr改好提交合并。

**Comment by liutaoyibeizhanyong** [comment]
麻烦帮忙添加一下河南省郑州市的金科星澜园

**Reply by WeihanLi** [reply]
直接 PR？

**Reply by yalo-ddaa** [reply]
直接修改然后发pr

**Comment by liutaoyibeizhanyong** [comment]
麻烦给了git权限 我是前端会git的 然后我最近也比较闲 可以为老百姓出一份力

**Comment by honthion** [comment]
我可以，熟悉git操作

**Comment by kztj** [comment]


**Reply by liutaoyibeizhanyong** [reply]
麻烦帮忙添加一下河南省郑州市的金科星澜园

**Comment by huhugit** [comment]
愿意出一份力

**Reply by huhugit** [reply]
也是烂尾楼受害者之一

**Comment by AnZhiYu** [comment]
我可以的

**Comment by brucemeteor** [comment]
> 是否有人愿意参与合并分支等？可以在此留言，给权限，合并新的提交以及校验数据是否正确，其他切勿修改
> 
> 首先你github总的玩的溜啊，别把文档合并坏了

im a developer and maybe i can help

**Comment by zhhrz** [comment]
建议 PR 添加一些 labels? 比如

- conflict
- need-image
- off-topic/not-suspension

**Reply by zhhrz** [reply]
我可以帮忙合并

**Comment by zhongweikang** [comment]
我可以

**Comment by wongzhe** [comment]
可以




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "WeNeedHome/SummaryOfLoanSuspension"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年7月15日(星期五) 中午1:25
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [WeNeedHome/SummaryOfLoanSuspension] 是否有人愿意参与合并分支等？可以在此留言，给权限，合并新的提交以及校验数据是否正确，其他切勿修改 (Discussion #423)





 
我也愿意，星星之火。

**Comment by xiaer93** [comment]
我可以，星星之火可以燎原。

**Comment by ssuperlilei** [comment]
我可以，看我GitHub提交记录，绝对溜

**Comment by hunao0221** [comment]
我可以，白天有时间.

**Comment by Jesse3692** [comment]
我白天晚上都可以的

**Comment by GuoyiZhang** [comment]
给我一个权限吧，我来操作，熟悉git各种操作

**Comment by GuoyiZhang** [comment]
给我一个权限吧，我来操作，熟悉git各种操作

**Comment by GuoyiZhang** [comment]
给我一个权限吧，我来操作，熟悉git各种操作

**Comment by GuoyiZhang** [comment]
给我一个权限吧，我来操作，熟悉git各种操作

**Comment by GuoyiZhang** [comment]
给我一个权限吧，我来操作，熟悉git各种操作

**Comment by ethanrao** [comment]
美西 Pacifica Time Zone 
身边有朋友收到波及 可以负责中国时间夜间维护

**Comment by GuoyiZhang** [comment]
给我一个权限吧，我来操作，熟悉git各种操作

**Comment by GuoyiZhang** [comment]
给我一个权限吧，我来操作，熟悉git各种操作

**Comment by GuoyiZhang** [comment]
给我一个权限吧，我来操作，熟悉git各种操作，不会发表任何言论

**Comment by GuoyiZhang** [comment]
当前处理速度太慢了

**Comment by GuoyiZhang** [comment]
楼主在不在？

**Comment by yalo-ddaa** [comment]
楼主辛苦了，我的家人也曾经是烂尾楼的受害者之一，希望能帮忙。
经常用git，近期也没什么事，都可以帮忙。
项目的pr积攒的还是比较多的，我可以帮忙把未完善好的pr修改一下并提交合并。

**Reply by ghost** [reply]
请问 给了权限以后 可以帮我的专楼置顶吗 非常感谢

**Comment by dyg411822** [comment]
楼主辛苦了，给我个提交合并权限

**Comment by GuoyiZhang** [comment]
楼主，我是烂尾楼业主，给我一个权限吧，我来操作，熟悉git各种操作，不会发表任何言论。工作比较自由，随时合并代码。希望尽快同意

**Comment by GuoyiZhang** [comment]
楼主，我是烂尾楼业主，给我一个权限吧，我来操作，熟悉git各种操作，不会发表任何言论。工作比较自由，随时合并代码。希望尽快同意

**Reply by GuoyiZhang** [reply]
楼主，我是烂尾楼业主，给我一个权限吧，我来操作，熟悉git各种操作，不会发表任何言论。工作比较自由，随时合并代码。希望尽快同意

**Reply by GuoyiZhang** [reply]
楼主，我是烂尾楼业主，给我一个权限吧，我来操作，熟悉git各种操作，不会发表任何言论。工作比较自由，随时合并代码。希望尽快同意

**Reply by GuoyiZhang** [reply]
楼主，我是烂尾楼业主，给我一个权限吧，我来操作，熟悉git各种操作，不会发表任何言论。工作比较自由，随时合并代码。希望尽快同意

**Reply by WeNeedHome** [reply]
已经邀请，切勿发送过激言论，只做merge

**Reply by Cegazhos** [reply]
該更新下rp了 似乎只有你在活躍了

**Comment by yalo-ddaa** [comment]
楼主辛苦了，我的家人也曾经是烂尾楼的受害者之一，希望能帮忙。
经常用git，近期也没什么事，都可以帮忙。
我可以帮忙修改原先的PR并提交merge。
不会发表涉及政治的引战言论。

**Comment by robothot** [comment]
@WeNeedHome 我可以帮忙进行 pr 审查， 但是如何进行验证 pr 的数据准确性？

**Comment by tanjunwen0302** [comment]
我愿意，会使用git，不会乱发表言论

**Comment by enginin22** [comment]
sure; I would like to contribute it (not physically in China now)

**Comment by hugetiny** [comment]
国内可能没有wiki共同协作这个概念，github是自带wiki的。对于维护文档来说，不需要git 的门槛。
进项目的Settings->Features->Wikis 把你项目的Restrict editing to users in teams with push access only √去掉。
然后wiki就公开编辑了，设置一部分项目成员，即使有人捣乱，也可以立刻屏蔽，并改回。

**Comment by xujianliu** [comment]
我愿意，我程序员，会git，会merge。理性维权，不反国反党。

**Comment by LYM1029** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by LYM1029** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by LYM1029** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
